### PR TITLE
WT-6653 In the rollback to stable test, rollback/restart txn before retry.

### PR DESCRIPTION
**Pull request by ddanderson**
Group entire iteration of updates in a function that can be repeated after rollback.

### CODE REVIEW

**Comment by ddanderson** [comment]
@kommiharibabu, can you please review?  This is the same problem found in #5978, I think a more complete solution.

File: test/suite/test_rollback_to_stable10.py [hunk 1]
```diff
@@ -271,12 +276,17 @@ def test_rollback_to_stable_prepare(self):
...
             self.conn.set_timestamp('stable_timestamp=' + timestamp_str(50))
 
-        # Here's the update operation we'll perform, encapsulated so we can easily retry
+        # Here's the update operations we'll perform, encapsulated so we can easily retry
         # it if we get a rollback. Rollbacks may occur when checkpoint is running.
-        def simple_update(cursor, key, value):
-            cursor.set_key(key)
-            cursor.set_value(value)
-            self.assertEquals(cursor.update(), 0)
+        def range_updates(session, cursor, ds, value, nrows, prepare_config):
```
Comment (kommiharibabu) [inline test/suite/test_rollback_to_stable10.py:281]: can we name it as `prepare_range_updates`? as this function prepares the transaction.

**Review by kommiharibabu — APPROVED** [review]
lgtm

File: test/suite/test_rollback_to_stable10.py (hunk 1 above)
Comment (ddanderson) [reply test/suite/test_rollback_to_stable10.py]: Done.

**Pull request merged**